addons/l10n_fr_invoice_addr/models/account_move.py

```diff
@@ -0,0 +1,15 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = 'account.move'
+
+    l10n_fr_is_company_french = fields.Boolean(compute='_compute_l10n_fr_is_company_french')
+
+    @api.depends('company_id.country_code')
+    def _compute_l10n_fr_is_company_french(self):
+        for record in self:
+            # France & DOM-TOM
+            record.l10n_fr_is_company_french = record.company_id.country_code in ['FR', 'MF', 'MQ', 'NC', 'PF', 'RE', 'GF', 'GP', 'TF']
```

Comment:
don't we want to use the method i talked last time like record.l10n_fr_is_company_french = ....